content/rosa/gpu/index.md

```diff
@@ -44,7 +43,7 @@ brew install jq
 
 ### Helm Prerequisites
 
-If you plan to use Helm to deploy the GPU operator, you will need to do the following
+> If you do not want to use Helm you can follow the steps in the [Manual](./manual) section.
```

Comment:
manual links here is broken, and there's no manual links in the main mobb.ninja page